src/middleware.ts

```diff
@@ -0,0 +1,7 @@
+// Without a defined matcher, this line applies next-auth to the entire project
+export { default } from 'next-auth/middleware';
+
+// Pages that the user cannot visit unless they login
+export const config = {
+  matcher: ['/', '/capture', '/maps', '/settings'],
```

Comment:
add  /editaccount

Reply:
those are sub pages of settings, so the URL would look like these for the edit account pages

`settings/edit-email`
`settings/change-password`

Reply:
with the current code, everything under `settings/` will naturally be a protected page